src/EnergyPlus/WindowManager.cc

```diff
@@ -2273,7 +2263,7 @@ namespace Window {
             if (ANY_SHADE_SCREEN(ShadeFlag) || ANY_BLIND(ShadeFlag)) {
                 IConst = s_surf->SurfWinActiveShadedConstruction(SurfNum);
             }
-            TotLay = state.dataConstruction->Construct(IConst).TotLayers;
+            int TotLay = state.dataConstruction->Construct(IConst).TotLayers;
```

Comment:
The diffs and crash was right here when this was changed to use the shortcut from above. If there is a shade or blind then IConst changes as does the TotLay'ers.

![image](https://github.com/user-attachments/assets/5309c059-9bee-44a2-a43f-638221b0b563)